airflow/contrib/hooks/gcs_hook.py

```diff
@@ -109,3 +109,41 @@ def exists(self, bucket, object):
             if ex.resp['status'] == '404':
                 return False
             raise
+
+    def is_updated_after(self, bucket, object, ts):
+        """
+        Checks if an object is updated in Google Cloud Storage.
+
+        :param bucket: The Google cloud storage bucket where the object is.
+        :type bucket: string
+        :param object: The name of the object to check in the Google cloud
+            storage bucket.
+        :type object: string
+        :param ts: The timestamp to check against.
+        :type ts: datatime
+        """
+        service = self.get_conn()
+        try:
+            response = service \
+                .objects() \
+                .get(bucket=bucket, object=object) \
+                .execute()
+
+            if 'updated' in response:
+                import dateutil.parser
+                import pytz
+
+                utc = pytz.UTC
```

Comment:
actually, I'll remove it because Airflow is assumed to be running on a machine that is in UTC anyway